stable/localvolumeprovisioner/templates/configmap.yaml

```diff
@@ -6,11 +6,13 @@ metadata:
   namespace: {{ .Release.Namespace }}
 data:
   storageClassMap: |
-    localvolumeprovisioner:
-       hostDir: /mnt/disks
-       mountDir: /mnt/disks
-       blockCleanerCommand:
-         - "/scripts/shred.sh"
-         - "2"
-       volumeMode: Filesystem
-       fsType: ext4
+    {{- range .Values.storageclasses }}
```

Comment:
I understand we have tested this change. 
What would it happen with the old storage class `localvolumeprovisioner` ? what will it happen with the existing pods using the storage class ?

Reply:
Good point, I'll change this to make both provisioner name as well as mount dir name configurable. With that we can provide a default configuration with storage class `localvolumeprovisioner` and mount dir `/mnt/disks` that's backwards-compatible with existing deployments.

Reply:
Fixed in https://github.com/mesosphere/charts/pull/118.